Sources/Build/Command.compile(SwiftModule).swift

```diff
@@ -35,7 +35,6 @@ extension Command {
 
         let tool = SwiftcTool(module: module, prefix: prefix, otherArgs: args + otherArgs, executable: SWIFT_EXEC, conf: conf)
 
-        //FIXME these should be inferred as implicit inputs by llbuild
         let mkdirs = Set(tool.objects.map{ $0.parentDirectory }).map(Command.createDirectory)
```

Comment:
Err, this isn't right. The FIXME is about not needing to run these commands at all, the right resolution to that issue is to ditch the createDirectory call and make sure stuff still works.